gramps/gen/lib/grampstype.py

```diff
@@ -227,6 +239,17 @@ def serialize(self):
         """Convert the object to a serialized tuple of data."""
         return (self.__value, self.__string)
 
+    @classmethod
+    def __get_str(cls, value, string):
+        if value == cls._CUSTOM:
+            return string
+        else:
+            return cls._I2SMAP.get(value, _UNKNOWN)
+
+    @classmethod
+    def get_str(cls, dict):
+        return cls.__get_str(dict["value"], dict["string"])
```

Comment:
It is usually good to avoid using Python names as parameter names. We've been calling the dict representation "data" in other places. How about:

```suggestion
    def get_str(cls, data):
        return cls.__get_str(data["value"], data["string"])
```

Or even:

```suggestion
    def get_str(cls, data):
        return cls.__get_str(data.value, data.string)
```
with the new DataDict wrapper.